lib/mix/torch.ex

```diff
@@ -70,6 +70,15 @@ defmodule Mix.Torch do
        do: inject_templates(mix_task, format, to_string(phoenix_version))
 
   def inject_templates("phx.gen.html", format, phoenix_version) when is_binary(phoenix_version) do
+    if Version.match?(phoenix_version, ">= 1.5.0") do
```

Comment:
This check is not needed here.  We are only currently conditionally making a check on version here for a single file and we can easily just do the check inline in the `controller.ex` template like you originally had started to do in #169. 

```elixir
<%= if Version.match?(to_string(Application.spec(:phoenix, :vsn)), "< 1.5.0") do %>
plug(:put_root_layout, {<%= inspect context.web_module %>.LayoutView, "torch.html"})
<% else %>
plug(:put_layout, {<%= inspect context.web_module %>.LayoutView, "torch.html"})
<% end %>
```

In fact, I'm going to make this same change to remove the duplicate phx1_5 variant we have for the layout file as well.  It's unneeded and we don't need to carry along a 2nd file just for these simple one-line conditionals in our templates.  I'll make the change for the existing layout file and make another PR for that.